src/components/atoms/Selection/Selection.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="w-[210px]">
+    <label class="text-left text-black w-[100%] truncate" v-if="props.selection.label">
+      {{ props.selection.label }}
+      <span v-if="props.selection.asterisk" class="text-red-500">*</span>
+    </label>
+
+    <div :class="['cursor-pointer', { 'pointer-events-none opacity-50': props.selection.isDisabled }]">
+      <!-- selected -->
+      <div
+        class="flex justify-between items-center w-[100%] h-[42px] px-[10px] border-[1px] border-gray-300 rounded-md"
+        @click="handleIsOpen"
+      >
+        <span v-if="selectedValue?.value">{{ selectedValue.label || props.selection?.valueList[0]?.label }}</span>
+        <div class="duration-300 " :class="{ '-rotate-180': isOpen }">
```

Comment:
```suggestion
<div class="duration-300" :class="{ '-rotate-180': isOpen }">
``` 
-> ko để dư space như này